src/uu/test/src/test.rs

```diff
@@ -128,23 +130,18 @@ pub fn uumain(mut args: impl uucore::Args) -> UResult<()> {
         }
     }
 
-    let result = parse(args).and_then(|mut stack| eval(&mut stack));
+    let result = parse(args).map(|mut stack| eval(&mut stack))??;
```

Comment:
Is this equivalent to `eval(parse(args)?)?`

Reply:
@tertsdiepraam Yes it is you prefer `eval(parse(args)?)?` syntax ?